crown-court-contribution/src/main/resources/db.changelog/changeset/02-correspondence-templates-table-create.sql

```diff
@@ -0,0 +1,12 @@
+--liquibase formatted sql
+--changeset muthus:02-correspondence-templates-table-create
+CREATE TABLE IF NOT EXISTS crown_court_contribution.CORRESPONDENCE_TEMPLATES
```

Comment:
I'm not very familiar with Liquibase so maybe this is not needed!? I can't see any indexes defined on any of the fields on any of the new tables, do we need to define indexes in these scripts or are they defined elsewhere?
(same feedback applies equally to other new tables defined in this PR also)

Reply:
These are just very small static metadata tables and we don't need Indexes